src/modules/facility-fixed-fee/facility-fixed-fee.controller.ts

```diff
@@ -0,0 +1,44 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { ApiBadRequestResponse, ApiInternalServerErrorResponse, ApiOkResponse, ApiOperation } from '@nestjs/swagger';
+
+import { GetFacilityFixedFeeParamsDto } from './dto/get-facility-fixed-fee-params.dto';
+import { GetFacilityFixedFeeResponse, GetFacilityFixedFeeResponseItem } from './dto/get-facility-fixed-fee-response.dto';
+import { FacilityFixedFeeService } from './facility-fixed-fee.service';
+
+@Controller()
+export class FacilityFixedFeeController {
+  constructor(private readonly facilityFixedFeeService: FacilityFixedFeeService) {}
+  @Get('/facilities/:facilityIdentifier/fixed-fees')
+  @ApiOperation({
+    summary: 'Get all fixed fees for a facility.',
+  })
+  @ApiOkResponse({
+    description: 'The fixed fees for the facility have been successfully retrieved.',
+    type: GetFacilityFixedFeeResponseItem,
+    isArray: true,
+  })
+  @ApiBadRequestResponse({
+    description: 'The specified facilityIdentifier is not valid.',
+  })
+  @ApiInternalServerErrorResponse({
+    description: 'An internal server error has occurred.',
+  })
+  async getFixedFeesForFacility(@Param() params: GetFacilityFixedFeeParamsDto): Promise<GetFacilityFixedFeeResponse> {
+    const fixedfeesForFacility = await this.facilityFixedFeeService.getFixedFeesForFacility(params.facilityIdentifier);
+    return fixedfeesForFacility.map((fixedFee) => ({
+      facilityIdentifier: fixedFee.facilityIdentifier,
+      portfolioIdentifier: fixedFee.portfolioIdentifier,
+      amount: fixedFee.amount,
+      effectiveDate: fixedFee.effectiveDate,
+      expirationDate: fixedFee.expirationDate,
+      nextDueDate: fixedFee.nextDueDate,
+      nextAccrueToDate: fixedFee.nextAccrueToDate,
+      period: fixedFee.period,
+      description: fixedFee.description,
+      currency: fixedFee.currency,
+      lenderTypeCode: fixedFee.lenderTypeCode,
+      incomeClassCode: fixedFee.incomeClassCode,
+      spreadToInvestorsIndicator: fixedFee.spreadToInvestorsIndicator,
```

Comment:
The dev team recently decided that this one-to-one mapping is unnecessary. For the time being, we're not worrying about removing it from endpoints that have already been merged in, but we're avoiding including it going forward. If you have time, it would be great if you could remove it from this endpoint, but if you think that will make the difference between you getting this merged in this sprint, I'd say don't worry about it for now.